csrc/fused_dense.cpp

```diff
@@ -62,7 +62,7 @@ std::vector<at::Tensor> linear_bias_backward(at::Tensor input, at::Tensor weight
 
   // create output/workspace tensor
   auto d_weight = at::empty({out_features, in_features}, input.type());
-#if defined(CUDA_VERSION) && CUDA_VERSION < 11000
+#if defined(CUDA_VERSION) && CUDA_VERSION < 11600
```

Comment:
I assume you would like to guard against CUDA11.4 (if so, then you would have to use `11040`) or against the `CUBLAS_VERSION`?

Reply:
Updated it now. It is tricky to guard against CUDA version since different epilogue constants in CUBLAS are added at different versions while still using CUDA 11.4.

Reply:
Note: Compile-time guards checking for new features may still require runtime guards and fallbacks, since it's legal to build against a newer version than you run with.